Spoj1Divisor Summation/Program.cs

```diff
@@ -2,31 +2,74 @@
 using System.Collections.Generic;
 using System.IO;
 using System.Linq;
+using System.Security.Cryptography.X509Certificates;
 using System.Text;
 using System.Threading.Tasks;
 
 namespace Spoj1Divisor_Summation
 {
-    public class Program
-    {
-        public static void Main()
-        {
-            var reader = new StreamReader(Console.OpenStandardInput());
-            var writer = new StreamWriter(Console.OpenStandardOutput());
+    public class Program
+    {
+        public static List<int> Inputs = new List<int>();
+        public static List<int> Outputs = new List<int>();
+        public static void Main()
+        {
+            var reader = new StreamReader(Console.OpenStandardInput());
+            var writer = new StreamWriter(Console.OpenStandardOutput());
             writer.AutoFlush = false;
 
-            DivisorSumation(reader, writer);
+            DivisorSumation(reader, writer);
         }
-
         public static void DivisorSumation(TextReader reader, TextWriter writer)
         {
-           //*******************************
-           //* TODO: Implement method body *
-           //*******************************
+            //*******************************
+            //* TODO: Implement method body *
+            //*******************************
+            Console.WriteLine("Inputs");
+            TakeInputs(reader);
+
+            CalculateDividersSums();
+
+            Console.WriteLine("\nOutputs:");
+            for (int number = 0; number < Outputs.Count; number++)
+            {
+                writer.WriteLine(Outputs[number]);
+            }
 
             reader.Close();
             writer.Flush();
             writer.Close();
         }
+        private static void TakeInputs(TextReader reader)
+        {
+            bool isNumber = false;
+            while (!isNumber)

```

Comment:
tutaj  można użyć for'a sterowanego ilością danych podaną w pierwszej linijce input'u
Ciało funkcji można by wyciągnąć do oddzielnej metody z odpowiednia nazwą